auto-value-gson-runtime/src/main/java/com/ryanharter/auto/value/gson/internal/SerializableWrapper.java

```diff
@@ -0,0 +1,47 @@
+// modified by mapbox
+package com.ryanharter.auto.value.gson.internal;
+
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonElement;
+
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+import java.util.Objects;
+
+public class SerializableWrapper implements Serializable {
```

Comment:
Could you explain why do we need this class?

Reply:
`JsonObject` and other classes from gson aren't serialisable. If your model is `Serializable`, unrecognised fields breaks java serialisation. Serializable wrapper is a workaround to let unrecognised fields to be serialised.

Reply:
Renamed to `SerializableJsonElement`